.github/workflows/push_images.yml

```diff
@@ -0,0 +1,107 @@
+#    Therefore, this cannot be triggered until test.yml is complete. 
+# 2. Trigger by workflow dispatch.
+#    Specify the PR number with the artifacts you'd like to use.
+#    Again, the test.yml workflow must be complete.
+# 3. By commenting on the PR `please update screenshots`.
+#
+# If a PR has already been opened on the conda-store repo from this
+# action, a rerun of this action will attempt to update the PR, not
+# create a new one. 
+
+name: Push screenshots to conda-store docs
+
+on:
+  pull_request:
+    types: [labeled]
+  issue_comment:
+    types: [created, edited]
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: "PR Number with screenshots to push"
+        required: true
+
+jobs:
+  update_screenshots:
+    name: "update_screenshots"
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -el {0}
+    if: ${{ github.event.label.name == 'update_screenshots' && github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' || (github.event.issue.pull_request && contains(github.event.comment.body, 'please update screenshots')) }}
+    steps:
+      - name: "Checkout the conda-store repository 🛎"
+        uses: actions/checkout@v4
+        with:
+          path: conda-store-repo
+          repository: 'conda-incubator/conda-store'
+          
+      - name: Use this PR number or the one specified in workflow dispatch
+        run: |
+          if [[ ${{ github.event_name == 'workflow_dispatch' }} == true ]]; then
+            my_variable=${{ inputs.pr_number }}
+          else
+            PR_NUMBER=${{github.event.pull_request.number}}
+          fi
+            echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_ENV
+          
+      - name: Download artifact images - will fail if artifact doesn't exist yet
+        id: download-artifact
+        uses: dawidd6/action-download-artifact@v3
+        with:
+            #   Optional, GitHub token, a Personal Access Token with `public_repo` scope if needed
+            # Required, if the artifact is from a different repo
+            # Required, if the repo is private a Personal Access Token with `repo` scope is needed or GitHub token in a job where the permissions `action` scope set to `read`
+            github_token: ${{ secrets.GITHUB_TOKEN }}
+            # workflow file name from which to get the artifact
+            workflow: test.yml
+            # PR from which to get the artifact
+            pr: ${{ env.PR_NUMBER }}
+            # how to exit the action if no artifact is found
+            if_no_artifact_found: fail
+            # artifact name
+            name: 'screenshots'
+            # directory in which to put the contents of the artifact
+            path: 'screenshots'
+
+            
+      - name: "Copy images to conda-store repo"
+        run: |
+            cd conda-store-repo
+            cp ../screenshots/*.png docusaurus-docs/conda-store-ui/images/.
+            git config user.name ${{ github.actor }}
```

Comment:
Checking my understanding here. GitHub actor is one of:

- user that dispatched workflow
- creator of PR
- person who comments on the PR "please update screenshots"

Is this the standard way to do it? Is there no GitHub Actions bot user?? Just wondering if this is robust and standard. I'm not a GitHub Actions expert so I have no idea.

Reply:
github actor is either "user that dispatched workflow" or "creator of PR". For anyone commenting to trigger a rerun, it will run as the creator of PR.

You could put something arbitrary in here, but in my mind this is a way to track WHO triggered the changes which opened the PR on the other repo. The PR will be opened by Jaime's user because we had to have credentials to be able to open that PR. Of course, if we aren't interested in the "who", we can just change it to a fake username.

Reply:
done!

Reply:
oh wait, did you forget to push your changes? I don't see any change to the commit message. It still shows as "commit all changes" to me